### PR TITLE
change the units for Z1pt0

### PR DESCRIPTION
**Pull request by jasonmotha**

### CODE REVIEW

**Comment by jasonmotha** [comment]
@lee-robin Are there other changes / bugs we have to address in the empirical models before doing more computation?

File: empirical/util/openquake_wrapper.py [hunk 1]
```diff
@@ -176,6 +177,8 @@ def oq_run(model, site, fault, im, period=None, **kwargs):
     if len(extra_site_parameters) > 0:
         raise ValueError("unknown site property: " + extra_site_parameters)
     oq_site = check_properties(site, model, SITE_PROPERTIES, oq_site, np_array=True)
+    if hasattr(oq_site, "z1pt0"):
+        oq_site.z1pt0 *= 1000
```
Comment (SnowNooDLe) [inline empirical/util/openquake_wrapper.py:181]: Is this just for Z1.0? or should we need one for Z2.5 as well?

Comment (jasonmotha) [reply empirical/util/openquake_wrapper.py:181]: We use km for all Z values in our Z extraction and in Empirical Engine.

OpenQuake use m for Z1pt0 and km for Z2pt5. And empirical models use whichever they feel like.

Comment (SnowNooDLe) [reply empirical/util/openquake_wrapper.py:181]: Nice consistency, m and km haha :joy:

**Review by SnowNooDLe — APPROVED** [review]

**Review by SnowNooDLe — APPROVED** [review]

**Review by jpa198 — APPROVED** [review]

**Review by SnowNooDLe — APPROVED** [review]

**Review by lee-robin — APPROVED** [review]

**Pull request merged**